site/en/r2/tutorials/distribute/keras.ipynb

```diff
@@ -32,284 +45,304 @@
+        "  </td>\n",
+        "</table>"
       ]
     },
     {
-      "cell_type": "markdown",
       "metadata": {
         "colab_type": "text",
         "id": "xHxb-dlhMIzW"
       },
+      "cell_type": "markdown",
       "source": [
         "## Overview\n",
         "\n",
-        "The `tf.distribute.Strategy` API is an easy way to distribute your training\n",
-        "across multiple devices/machines. Our goal is to allow users to use existing\n",
-        "models and training code with minimal changes to enable distributed training.\n",
+        "The `tf.distribute.Strategy` API provides an abstraction for distributing your training\n",
+        "across multiple processing units. The goal is to allow users to enable distributed training using existing models and training code, with minimal changes.\n",
         "\n",
-        "Currently, core TensorFlow includes `tf.distribute.MirroredStrategy`. This\n",
+        "This tutorial uses the `tf.distribute.MirroredStrategy`, which\n",
         "does in-graph replication with synchronous training on many GPUs on one machine.\n",
-        "Essentially, it create copies of all variables in the model's layers on each\n",
-        "device. It then use all-reduce to combine gradients across the devices before\n",
-        "applying them to the variables to keep them in sync.\n",
+        "Essentially, it copies all of the model's variables to each processor.\n",
+        "Then, it uses [all-reduce](link to explanation) to combine the gradients from all processors and applies the combined value to all copies of the model.\n",
         "\n",
-        "Many other strategies will soon be\n",
-        "available in core TensorFlow. You can find more information about them in the\n",
-        "[README](https://github.com/tensorflow/tensorflow/tree/master/tensorflow/contrib/distribute).\n",
+        "`MirroredStategy` is one of several distribution strategy available in TensorFlow core. Advanced users can implement the `tf.distribute.Strategy` API to create custom strategies.\n",
+        "For more details, see the [Distribution Strategy README](https://github.com/tensorflow/tensorflow/tree/master/tensorflow/contrib/distribute).\n",
         "\n"
       ]
     },
     {
-      "cell_type": "markdown",
       "metadata": {
         "colab_type": "text",
         "id": "MUXex9ctTuDB"
       },
+      "cell_type": "markdown",
       "source": [
-        "## Example with Keras API\n",
+        "### Keras API\n",
         "\n",
-        "The easiest way to get started with multiple GPUs on one machine using `MirroredStrategy` is with `tf.keras`."
+        "To keep this example as clear as possible, we'll use the `tf.keras` API to build the model and training loop. (For an advanced example using a custom training loop, see [this tutorial](/tutorials/distribute/training_loops))."
       ]
```

Comment:
Use relative paths. Or, full URLs. Absolute paths `/like/this` are never the right thing.
Always include extensions in the source files (except .yaml files), these are stripped during the publishing process.